--- ./android_configure.py	2025-04-04 22:18:44
+++ ./android_configure2.py	2025-04-04 22:21:00
@@ -74,4 +74,4 @@
 os.environ['GYP_DEFINES'] = GYP_DEFINES
 
 if os.path.exists("./configure"):
-    os.system("./configure --dest-cpu=" + DEST_CPU + " --dest-os=android --openssl-no-asm --cross-compiling")
+    os.system("./configure --dest-cpu=" + DEST_CPU + " --dest-os=android --openssl-no-asm --cross-compiling --without-inspector --shared")
