--- ./deps/zlib/zlib.gyp	2025-04-05 01:52:11
+++ ./deps/zlib/zlib2.gyp	2025-04-05 02:21:50
@@ -196,6 +196,9 @@
                 'USE_FILE32API'
               ],
             }],
+            ['OS=="android"', {
+              'dependencies': [ 'libcpufeatures' ],
+            }],
             # Incorporate optimizations where possible.
             ['(target_arch in "ia32 x64" and OS!="ios") or arm_fpu=="neon"', {
               'dependencies': [ 'zlib_inflate_chunk_simd' ],
@@ -265,5 +268,23 @@
         },
       ],
     }],
+    ['OS=="android"', {
+      'targets': [
+        {
+          'target_name': 'libcpufeatures',
+          'type': 'static_library',
+          'sources': [
+            '<!(node -p "path.relative(path.resolve(\'deps/zlib\'),path.resolve(process.argv[1],\'sources/android/cpufeatures/cpu-features.c\')).replace(process.argv[2],\'/\')" "<(android_ndk_path)" "\\\\")'
+            # '<(android_ndk_path)/sources/android/cpufeatures/cpu-features.c',
+          ],
+          'cflags': [ '-Wall', '-Wextra', '-Werror' ],
+          'link_settings': {
+            'libraries': [
+              '-ldl',
+            ]
+          }
+        }
+      ],
+    }],
   ],
 }
